0000-payment-pointers/0000-payment-pointers.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Payment Pointers
+draft: 1
+---
+# Payment Pointers and Payment Setup Protocols
+
+## Abstract
+
+Payment pointers are a standardized identifier for accounts that are able to recieve payments. In the same way that an email address provides an identifier for a mailbox in the email ecosystem a payment pointer is used by an account holder to share the details of their account with anyone that wishes to make a payment to them.
+
+A payment pointer can be normalized to an "https" URL that provides the location of a payment setup service endpoint at which a sender can initiate a payment to the receiver.
+
+## Design Goals
+
+The following design goals are met by this specification:
+
+### Unique and Easily Recognizable
+
+Various standardized and defacto-standardized identifiers are widely used on the Internet today such as email addresses and social media handles. The payment pointer must be obviously unique so that it is not confused with another type of identifier and also so that other identifiers are not assumed to be payment pointers.
+
+### Simple Transcription
+
+It should be easy for someone to exchange their payment pointer with a potential payee either by saying it, writing it down or sending it in a digital message.
+
+### Flexible
+
+The payment pointer should not be tightly couple to a specific payment setup protocol or protocol version. It should be possible for any new payment setup protocol to leverage payment pointers.
+
+### Widely Usable
+
+It should be simple for individuals or small companies to host their own payment setup protocol receiver endpoints at a URL that can be resolved via a simple and easily recognizable payment pointer. Likewise, it should be possible for a payment services provider to host payment setup protocol receiver endpoints for multiple entities without the risk of hosting them at endpoint URLs that conflict with their other services. To that end the provider should have the option of hosting different receivier endpoints under the same domain and a different path or at a different subdomain for each receiver.
+
+## Payment Setup Protocols
+
+A payment setup protocol is defined very generically as a protocol for exchanging payment information between a sender and receiver before executing the payment.
+
+An example of a payment setup protocol is the [Simple Payment Setup Protocol](https://interledger.org/rfcs/0009-simple-payment-setup-protocol/) used to setup an Interledger payment.
+
+Payment pointers MUST resolve to an endpoint hosted by a payment receiver at which a payment setup protocol can be initiated. This endpoint is known as the payment setup protocol receiver endpoint.
+
+A payment setup protocol receiver endpoint is identified by an https URL as defined in [RFC7230](https://tools.ietf.org/html/rfc7230#section-2.7.2). It MUST accept HTTP GET requests to initiate the protocol.
```

Comment:
Wouldn't whether it responds to GET requests depend on the setup protocol used?

Reply:
No, that needs to be a minimal requirement because that kicks of protocol negotiation

Reply:
We could add DNS SRV based discovery but I think that would be good as a fall-back we can add later if there are setup protocols people want to introduce that are not HTTP based

Reply:
I was thinking more about protocols that might use a POST or another HTTP call instead of a GET